ui-tests/src/test/java/org/openmrs/reference/page/HtmlFormsPage.java

```diff
@@ -0,0 +1,36 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.Person;
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class HtmlFormsPage extends Page {
+    private ManageFormsPage manageFormsPage;
+    private HomePage homePage;
+    private static final By NEW_HTML_FORM = By.cssSelector("a[href='htmlForm.form']");
+    private static final By NAME = By.name("form.name");
+    private static By DESCRIPTION = By.cssSelector("#content textarea[name='form.description']");
+    private static  By SAVE_FORM = By.cssSelector("input[type=\"submit\"]");
+    private  static By VERSION = By.name("form.version");
+
+    public HtmlFormsPage(ManageHtmlFormsPage page) {
+        super(page);
+    }
+
+    public HtmlFormsPage(Page page) {
+        super(page);
+    }
+
+    public  void createNewTestForm(String name, String description, String version){
+        setTextToFieldNoEnter(NAME, name);
+        setTextToFieldNoEnter(DESCRIPTION,description);
+        setTextToFieldNoEnter(VERSION,version);
+    }
+    public void saveForm() throws InterruptedException {
+            clickOn(SAVE_FORM);
```

Comment:
once again please ensure you first check if the form doesn't existing on the page to add it.

Reply:
This functionality adds a form within the htmlformsPage and when you give a check, so am still skeptical how this is gona jump into manageForm from configure metadata, May be creating another method that checks whether form is present